src/search.cpp

```diff
@@ -295,6 +295,7 @@ void Thread::search() {
   beta = VALUE_INFINITE;
 
   if (mainThread)
+      mainThread->fadeBestMoveChanges = 0;
```

Comment:
missing '''{ }''' ?

Reply:
Oh yeah. :-(
That's really annoying because I made exactly the same mistake a day or two ago and took a while to figure out what I'd done wrong, so I should have noticed that easily.

Reply:
@vondele without the "{}" previously only the initialization of `mainThread->fadeBestMoveChanges = 0;` wud run when the if condition on line 297 becomes true, but with brackets, the for loop on line 300 will also run when and only if the if condition is true. Previously for loop wud have run always irrespective of the if condtion on line 297 being true or false. Doesn't this change the logic of the original passed patch vs this change and shud that be tested again? Please confirm.